base/lnls-get-n-unpack.sh

```diff
@@ -19,6 +19,14 @@ for url; do
 
     echo Downloading "$url"...
     wget -P $download_dir -o /tmp/wget.log "$url" || (cat /tmp/wget.log && false)
-    tar --no-same-owner -xf $download_dir/* -C $dest
+
+    filename=$(basename $download_dir/*)
+
+    if [[ ${filename,,} == *".zip" ]]; then
+        unzip -qo $download_dir/$filename -d $dest
+    else
+        tar --no-same-owner -xf $download_dir/$filename -C $dest
+    fi
```

Comment:
I think if we end up having to support more formats, we should switch to bsdtar+libarchive directly, instead of having to handle different tools in the script. Since busybox already provides unzip, I think it's reasonable to depend on it.